src/strtree.h

```diff
@@ -3,22 +3,32 @@
 
 #include <Python.h>
 #include "geos.h"
+#include "pygeom.h"
 
 
-typedef struct {
-    PyObject_HEAD
-    void *ptr;
-    PyObject *geometries;
-    long count;
-} STRtreeObject;
-
 /* A resizable vector with numpy indices */
 typedef struct
 {
     size_t n, m;
     npy_intp *a;
 } npy_intp_vec;
 
+
+/* A resizable vector with pointers */
+typedef struct
+{
+    size_t n, m;
+    GeometryObject **a;
+} goes_geom_vec;
```

Comment:
```suggestion
} geom_obj_vec;
```

? (it are not GEOSGeometries that get stored)

Reply:
And a typo in geos ;) I renamed it.